webapp/src/js/controllers/send-message.js

```diff
@@ -28,7 +29,7 @@ angular.module('inboxControllers').controller('SendMessageCtrl',
       if (message) {
         $scope.error.message = false;
       } else {
-        return translateRequiredField('tasks.0.messages.0.message')
```

Comment:
You've done one, but there are still more in this file. Is there a reason that you're only doing this one and not the other?

Reply:
Yes, I missed this one. Will fix.